security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisTool.java

```diff
@@ -214,7 +213,25 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
 
         String currentState = network.getVariantManager().getWorkingVariantId();
 
-        SecurityAnalysisResult result = securityAnalysis.run(currentState, parameters, contingenciesProvider).join();
+        SecurityAnalysisResult result;
+        if (!line.hasOption(OUTPUT_LOG_OPTION)) {
```

Comment:
Note: you may use the ToolOptions utility class, which wraps command line options in a typed Optional (Optional\<Path\> here for example).